### PR TITLE
Fix: To check if kebechet's advise manager is working on all overlays

### PR DESCRIPTION
**Pull request by shreekarSS**
## Related Issues and Dependencies
Fixes: https://github.com/thoth-station/ps-nlp/issues/151
Fixes: https://github.com/thoth-station/support/issues/247
Fixes: https://github.com/thoth-station/support/issues/248
Fixes: https://github.com/thoth-station/kebechet/issues/1094
Fixes: https://github.com/thoth-station/support/issues/251
…

## Description

Fixed referencing incorrect dict keys and Piplock file path

### CODE REVIEW

File: kebechet/managers/thoth_advise/thoth_advise.py [hunk 1]
```diff
@@ -380,16 +377,22 @@ def run(self, labels: list, analysis_id=None):
...
                 self.runtime_environment = _runtime_env_name_from_advise_response(
                     res[0]
                 )
                 to_ret = False
                 if res[1] is False:
                     _LOGGER.info("Advise succeeded")
                     self._write_advise(res[0])
+                    res[0].update({"document_id": analysis_id})
+                    file_path = os.path.join(
+                        overlays_dir, str(self.runtime_environment), "Pipfile.lock"
```
Comment (KPostOffice) [inline kebechet/managers/thoth_advise/thoth_advise.py:392]: Hey, what happens if there is no overlays dir? `overlays_dir` will still be an empty string. This should looks something like:
```
file_path = os.path.join(overlays_dir, self.runtime_environment, "Pipfile.lock") if overlays_dir else "Pipfile.lock"
```

Comment (shreekarSS) [reply kebechet/managers/thoth_advise/thoth_advise.py:392]: Ack, thanks for pointing out

File: kebechet/managers/thoth_advise/thoth_advise.py [hunk 2]
```diff
@@ -380,16 +377,26 @@ def run(self, labels: list, analysis_id=None):
...
+                    file_path = (
+                        os.path.join(
+                            overlays_dir, str(self.runtime_environment), "Pipfile.lock"
+                        )
+                        if overlays_dir
+                        else "Pipfile.lock"
+                    )
                     opened_merge = self._open_merge_request(
-                        branch_name, labels, ["Pipfile.lock"], res[0].get("metadata")
+                        branch_name, labels, [file_path], res[0]
```
[the remark this answers is not in the document]
Comment (shreekarSS) [reply kebechet/managers/thoth_advise/thoth_advise.py:399]: I'm just taking document_id from the dict, https://github.com/shreekarSS/kebechet/blob/b405c224bea8763c94699c180ecf66b2aac18791/kebechet/managers/thoth_advise/thoth_advise.py#L143

**Pull request merged**